halyard-web/lib/systemd/system/halyard.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=A tool for managing the lifecycle of a Spinnaker installation
+
+[Service]
+Restart=always
+RestartSec=10
+Type=forking
+ExecStart=
```

Comment:
This isn't a mistake, it clears this var since it can be assigned multiple times.